tests/utest/oxygen/test_oxygen_cli.py

```diff
@@ -54,10 +55,20 @@ def test_junit_works_on_cli(self):
                      text=True,
                      shell=True)
 
-        for expected in ('<stat pass="69" fail="3">Critical Tests</stat>',
-                         '<stat pass="69" fail="3">All Tests</stat>'):
-            self.assertIn(expected, example.read_text())
-            self.assertIn(expected, actual.read_text())
+        exampleTree = ElementTree.parse(example)
+        exampleRoot = exampleTree.getroot()
+
+        actualTree = ElementTree.parse(actual)
+        actualRoot = actualTree.getroot()
+
+        # RF<4 has multiple `stat` elements therefore we use the double slash
+        # and a filter to single out the `stat` element with text "All Tests"
+        exampleStat = exampleRoot.find(
+            './statistics/total//stat[.="All Tests"]')
+        actualStat = actualRoot.find('./statistics/total//stat[.="All Tests"]')
```

Comment:
Let's not repeat the XPath pattern as it's the same for both:
```suggestion
        all_tests_stat_block = './statistics/total//stat[.="All Tests"]'
        expected_stat = exampleRoot.find(all_tests_stat_block)
        actual_stat = actualRoot.find(all_tests_stat_block)
```
Also, I think most of the project, we use `snake_case` rather than `camelCase`. `unittest`'s asserts are the exception, which is supremely annoying :angry: